docs/conf.py

```diff
@@ -97,7 +96,7 @@
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
-html_logo = os.path.join(os.path.abspath('.'), 'figures', 'logo.png')
+html_logo = os.path.join(os.path.abspath('.'), 'figures', 'pysatnasa_logo.png')
```

Comment:
So, there's a better way to manage files and stuff in packages now.  See the stuff here: https://github.com/aburrell/apexpy/pull/123

Reply:
I'm not sure how to implement this in the docs.  I don't have a package to refer to